src/main/java/com/iridium/iridiumskyblock/configs/Configuration.java

```diff
@@ -62,6 +69,32 @@
             .put(5, 17)
             .build();
 
+    /**
+     * The Rewards the island gets for leveling up
+     * The integer represents the reward they will get, if the island level is divisible by this number they will get that reward
+     * Higest number takes priority
+     * <p>
+     * e.g. 1 will give the reward to every level since every number is divisible by 1
+     * 5 will give the reward to levels 5 10 15 20 25 ect since they are divisible by 5
+     */
+    public Map<Integer, Reward> islandLevelRewards = ImmutableMap.<Integer, Reward>builder()
+            .put(1, new Reward(new Item(XMaterial.EXPERIENCE_BOTTLE, 1, "&b&lLevel %island_level% Reward", Arrays.asList(
+                    "&7Island Level %island_level% Rewards:",
+                    "&b&l* &b5 Island Crystals",
+                    "&b&l* &b1000 Island Money",
+                    "",
+                    "&b&l[!] &bLeft click to redeem"
+            )), Collections.emptyList(), 0, 5, 1000, 0, XSound.ENTITY_PLAYER_LEVELUP))
+
+            .put(5, new Reward(new Item(XMaterial.EXPERIENCE_BOTTLE, 1, "&b&lLevel %island_level% Reward", Arrays.asList(
+                    "&7Island Level %island_level% Rewards:",
+                    "&b&l* &b15 Island Crystals",
+                    "&b&l* &b10000 Island Money",
+                    "",
+                    "&b&l[!] &bLeft click to redeem"
+            )), Collections.emptyList(), 0, 15, 10000, 0, XSound.ENTITY_PLAYER_LEVELUP))
```

Comment:
I think that these default rewards are too high, you get to these levels pretty quickly if I recall correctly. 

I'd say 200 / 5 for level 1 and 2000 / 15 for level 5

Reply:
Considering the shop prices too. You can't give a new player enough money to buy some advanced stuff